.github/workflows/backend-ci.yaml

```diff
@@ -31,5 +31,7 @@ jobs:
       run: |
         black services/backend/api services/backend/tests --check
     - name: Test with pytest
+      env:
+        GEOCODER_API_KEY: ${{ secrets.GEOCODER_API_KEY }}
```

Comment:
FYI, this is how you add secrets to Github and then expose them in CI: https://docs.github.com/en/actions/security-guides/encrypted-secrets